### PR TITLE
Fix some typos in jkube-kit/common module 

### PR DESCRIPTION
**Pull request by nishantraut**
## Description

Fixed some typos in Javadocs

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Review by rohanKanojia — APPROVED** [review]

**Comment by rohanKanojia** [comment]
Looks good :+1: 

However, I have a minor nitpick regarding your PR/commit title. Please try to be as descriptive while writing PR/commit titles so that people who review can easily get an context regarding what they're going to review :pray: .

**Comment by nishantraut** [comment]
Noted. Thanks for the feedback.

File: jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/AsciiDocParser.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
      * Between line breaks, both columns representing the kind (first column) and filename (second column).
      * Notice that one-line columns definition @code{|Cell in column 1, row 1|Cell in column 1, row 1} is not supported.
      *
-     * This method returns an @code{IllegalArgumentException} if does not contain two columns.
+     * This method returns a @code{IllegalArgumentException} if it does not contain two columns.
```
Comment (manusa) [inline jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/AsciiDocParser.java:55]:
```suggestion
     * This method returns an {@link IllegalArgumentException} if it does not contain two columns.
```

Comment (nishantraut) [reply jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/AsciiDocParser.java]: Fixed suggested change.

**Review by manusa — APPROVED** [review]
LGTM, thx!

**Pull request merged**